### ISSUE TITLE
How to convert resnet101-based Mask-RCNN model to ONNX ?

### ISSUE THREAD

**Issue by pra-dan** [onnx/keras-onnx#716]
Hi, 

I have my my Mask-RCNN model weights (h5) and its based on ResNet101. I am unable to load my weights before converting them to ONNX. The example you provide, loads a ResNet50 weight 
```python3
# load keras model
from keras.applications.resnet50 import ResNet50
model = ResNet50(include_top=True, weights='imagenet')
```

There is no such utility for resnet101. How should I go forward ?
